using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -4,8 +4,160 @@
+
+#### Event Description
+
+This event has 2 purposes:
+
+1. Deliver the Audience Manager profile to the original requester
+2. Deliver the Audience Manager profile to any module that may be listening for updates.
+
+This event will be generated in the following scenarios:
+
+- When a request is made to update the Audience manager profile 
+- When a request is made to retrieve the current Audience manager profile 
+
+#### Event Details
+
+| Type                                | Source                                | Paired | Direction                                                    |
+| :---------------------------------- | :------------------------------------ | :----- | :----------------------------------------------------------- |
+| com.adobe.eventType.audienceManager | com.adobe.eventSource.responseContent | True   | This is the response to a paired event. This should be published both with and without a paired ID |
+
+#### Data Payload Definition
+
+Definition of the key/value pairs that will be present in this event
+
+| Key + Key Type         | Value Type | Optional | Description                                                  |
+| :--------------------- | :--------- | :------- | :----------------------------------------------------------- |
+| <CUSTOMER_KEY>(String) | String     | True     | The Data payload will be full of Customer key value pairs. The Data Payload will be populated using the CV and CN values in the AAM Stuff array |
+
+
+### Configuration Response Content
+
+#### Event Details
+
+| **Event Type**                    | **Event Source**                      | **Paired** | **Paired Event** |
+| :-------------------------------- | :------------------------------------ | :--------- | :--------------- |
+| com.adobe.eventType.configuration | com.adobe.eventSource.responseContent | No         | N/A              |
+
+The data property in the Configuration Response Content event is used by each extension to modify its current settings. Each extension is responsible to read out the part of the data property for which it is concerned.
+
+The Identity extension will read the following keys from the configuration event:
+
+Here are the key-value pairs in this event:
+
+| **Key**                  | **Value Type** | **Optional** | **Description**                                              |
+| :----------------------- | :------------- | :----------- | :----------------------------------------------------------- |
+| `experienceCloud.org`    | String         | Yes          | Experience Cloud Org Identifier                              |
+| `experienceCloud.server` | Sting          | Yes          | Custom endpoint to be used for Visitor ID Service network requests |
+
+
+### Event Hub Shared State
+
+#### Event Description
+
+The shared state event is dispatched by the event hub after a shared state is created or updated. The event data contains the event owner, the name of the module to which the shared state belongs.
```

Comment:
Please include details about why is the Identity extension listening for this event.